### PR TITLE
Adding post processing to InterProScan

### PR DESCRIPTION
**Pull request by jshleap**
Interproscan data requires some minor postprocessing. This pull request add said postprocessing to the recipe. It installs fine in the build node with the postprocessing.

### CODE REVIEW

File: easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+sources = ['%(namelower)s-%(version)s-64-bit.tar.gz']
+checksums = ['3b792b33a14091a144a13ba35d70fbaa45abb8102f240a6da4e24bb07babaac4']
+
+dependencies = [
+    ('Python', '3.7'),
+    ('HMMER', '3.2.1'),
+]
+
+
+postinstallcmds = [
```
Comment (ccoulombe) [inline easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb:28]: Please indent correctly

File: easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+'echo %(installdir)s && ls %(installdir)s',
+'export DATA_DIR=%(installdir)s; '
```
Comment (ccoulombe) [inline easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb]: No need to define it then overwrite it. You can use installdir directly

File: easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['data']
+}
+
+moduleclass = 'bio'
+
+
+# Build statistics
```
Comment (ccoulombe) [inline easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb]: No need for the stats in the recipe

File: easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+toolchain = SYSTEM
+source_urls = [
+    'http://ftp.ebi.ac.uk/pub/software/unix/iprscan/%(version_major)s/%(version)s/',
+]
+sources = ['%(namelower)s-%(version)s-64-bit.tar.gz']
+checksums = ['3b792b33a14091a144a13ba35d70fbaa45abb8102f240a6da4e24bb07babaac4']
+
+dependencies = [
+    ('Python', '3.7'),
```
Comment (ccoulombe) [inline easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb:23]: Does it only work with Python 3.7 or any Python?

If any python, then set a dependency on `python` in a modluafooter

**Review by ccoulombe — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb (hunk 4 above)
Comment (jshleap) [reply easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb:23]: Does that work with any python 3? I do not think is compatible with python2

File: easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+
+toolchain = SYSTEM
+source_urls = [
+    'http://ftp.ebi.ac.uk/pub/software/unix/iprscan/%(version_major)s/%(version)s/',
+]
+sources = ['%(namelower)s-%(version)s-64-bit.tar.gz']
+checksums = ['3b792b33a14091a144a13ba35d70fbaa45abb8102f240a6da4e24bb07babaac4']
+
+dependencies = [
```
Comment (jshleap) [inline easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb]: I changed it to builddependencies as these dependencies are only needed during building this datasets.

File: easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb (hunk 4 above)
Comment (jshleap) [reply easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb:23]: By changing the dependencies to `builddependencies` I believe this comment is resolved, correct?

Comment (ccoulombe) [reply easybuild/easyconfigs/i/InterProScan/InterProScan-5.50-84.0-data.eb:23]: Should be ok

**Comment by jshleap** [comment]
@ccoulombe Just pushed some changes

**Review by ccoulombe — APPROVED** [review]
Looks ok.

**Pull request merged**